packages/flutter/lib/src/material/button_bar.dart

```diff
@@ -12,7 +12,8 @@ import 'raised_button.dart';
 /// A horizontal arrangement of buttons.
 ///
 /// Places the buttons horizontally according to the padding in the current
-/// [ButtonTheme].
+/// [ButtonTheme]. Alignment and order of children are reversed between RTL and
```

Comment:
It might be best to be more specific here (since in one sense the Row's "alignment" doesn't change, it's always MainAxisAlignment.end).

The children are laid out in a [Row] with [MainAxisAlignment.end]. When the [Directionality] is [TextDirection.ltr] the button bar's children are aligned on the right and the last child becomes the rightmost child. When the [Directionality]  [TextDirection.rtl] the children are aligned on the left and the last child becomes the leftmost child.